pom.xml

```diff
@@ -115,15 +115,15 @@
     <geronimo-javamail_1.4_mail.version>1.9.0-alpha-2</geronimo-javamail_1.4_mail.version>
 
     <openjpa.version>3.1.0</openjpa.version>
-    <openwebbeans.version>2.0.12</openwebbeans.version>
+    <openwebbeans.version>2.0.15</openwebbeans.version>
```

Comment:
ticket mentions 2.0.16 and last release is 2.0.17 so maybe directly go to 2.0.17?

Reply:
I thought it would be easier to do it in two steps (at least for me), since 2.0.17 comes with the asm8 change. I was planning to do it once this one is finished

Reply:
Sorry, this should be 2.0.16, I will push this now